src/log.rs

```diff
@@ -24,15 +24,22 @@ byond_fn! { log_write(path, data) {
             Entry::Vacant(elem) => elem.insert(open(path)?),
         };
 
-        // write first line, timestamped
-        let mut iter = data.split('\n');
-        if let Some(line) = iter.next() {
-            write!(file, "[{}] {}\n", Utc::now().format("%F %T%.3f"), line)?;
-        }
+        if let Some(format_bool) = format.parse() {
```

Comment:
```suggestion
        if let Ok(format_bool) = format.parse() {
```

Reply:
although actually you should maybe deal with a possible error from the Result

Reply:
suggestion: `if let Ok(false) = format.parse()` to do without formatting, to both avoid the duplicate branch and to make the old behavior the default